### ISSUE TITLE
Discussion API (Language Model)

### ISSUE THREAD

**Issue by JacobLinCool** [hinagiku-dev/Hinagiku#29]
I think @TakalaWang already finished the main part of this, but I would like to have this issue open until the discussion room is done.

@TakalaWang can you share the timing and usage of those APIs?

**Comment by TakalaWang** [comment]
Of course, following is APIs for stage 1(Individual stage) and stage 2(group discussion stage). 

### Create conversation
**Method**: POST
**Endpoint**: /api/session/[id]/group/[group_number]/conversations/
**RequestBody**: {
	task: string
	subtasks: string[]
	resources: string[]
}

---

### Chat with LLM on conversation
**Method**: POST
**Endpoint**: /api/session/[id]/group/[group_number]/conversations/[conv_id]/chat/
**RequestBody**: {
	content: string
	audio: string // audio file id
}

---

### Summary chat history for student in stage 1
**Method**: GET
**Endpoint**: /api/session/[id]/group/[group_number]/conversations/[conv_id]/summary/

---

### Add one new discussion in group
**Method**: POST
**Endpoint**: /api/session/[id]/group/[group_number]/discussions/add/
**RequestBody**: {
	content: string
	speaker: string // speaker name not id
}

---

### Summary the group discussions
**Method**: GET
**Endpoint**: /api/session/[id]/group/[group_number]/discussions/summary/

---

### **Update the group discussions summary**
**Method**: PUT
**Endpoint**: /api/session/[id]/group/[group_number]/discussions/summary/
